app/templates/components/filter-option.hbs

```diff
@@ -0,0 +1,26 @@
+<form class="ui large form">
+  <div class="fields">
```

Comment:
If you use this, there will be no need of any custom styles. Try to use things provided by semantic